index.js

```diff
@@ -126,7 +126,7 @@ ${chalk.yellowBright.bold('Options: ')}
           fileSentences(data);
           // Get and display the absolute path of file.
           console.log(
-            `${chalk.blueBright.bold(
```

Comment:
provide matching highlight of text for `stats `
![image](https://user-images.githubusercontent.com/42750440/198055448-7453fed0-2515-4069-8fdb-5412e2c64421.png)

Reply:
![Screenshot from 2022-10-26 20-54-15](https://user-images.githubusercontent.com/84903954/198069289-9efc8184-eae4-4b7d-a215-a1fb27698d8c.png)
Done

Reply:
Also changed the white color to whiteBright color to match the system default for white text.